mava/components/jax/executing/base.py

```diff
@@ -46,7 +47,18 @@ def on_execution_init_start(self, executor: SystemExecutor) -> None:
         Returns:
             None.
         """
-        executor._interval = self.config.interval  # type: ignore
+        # TODO (Ruan): Double check that this is necessary
+        if self.config.evaluation_duration:
+            try:
```

Comment:
Change this to an if and raise ValueError exception, please :)

Reply:
Done :)

Reply:
Thanks :)